packages/paste-core/components/popover/src/Popover.tsx

```diff
@@ -25,8 +25,7 @@ const StyledPopover = React.forwardRef<HTMLDivElement, BoxProps>(({style, ...pro
       borderRadius="borderRadius20"
       boxShadow="shadowCard"
       maxWidth="size30"
-      minWidth="size30"
-      zIndex="zIndex10"
+      zIndex="zIndex90"
```

Comment:
Let's make this 80 since tooltip is 90 and should appear over popovers

Reply:
Sounds good.